src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -106,6 +106,51 @@ public class RuntimeConfigLoaderJsonDeserializerTests
             }
         }
 
+        /// <summary>
+        /// Tests the parsing of the runtime configuration with environment variables based on replaceEnvVar parameter.
+        /// </summary>
+        /// <param name="replaceEnvVar">A boolean indicating whether to replace environment variables in the configuration.</param>
+        [DataTestMethod]
+        [DataRow(false, DisplayName = "Do not replace environment variables.")]
+        [DataRow(true, DisplayName = "Replace environment variables.")]
+        public void TestConfigParsingWithEnvVarReplacement(bool replaceEnvVar)
```

Comment:
as mentioned in another comment, this test should have an additional param representing the config value because you'll need to test JSON converter behavior with MSSQL options as well as cosmos options.